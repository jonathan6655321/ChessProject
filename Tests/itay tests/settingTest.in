game_mode 2
print_setting
difficulty 2
print_setting
user_color 0
print_setting
default
print_setting
game_mode 1
print_setting
difficulty 4
print_setting
user_color 1
print_setting
start
reset
print_setting
game_mode 0
print_setting
difficulty 0
print_setting
user_color 2
print_setting
game_mode 1
print_setting
difficulty 3
print_setting
user_color 0
print_setting
default
print_setting
hello
game_mode
difficulty
user_color
game_mode 
difficulty 
user_color 
game_mode t
difficulty a
user_color b
quit