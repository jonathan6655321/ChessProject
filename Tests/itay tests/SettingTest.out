Specify game setting or type 'start' to begin a game with the current setting:
Game mode is set to 2 players
SETTINGS:
GAME_MODE: 2
Invalid command
SETTINGS:
GAME_MODE: 2
Invalid command
SETTINGS:
GAME_MODE: 2
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
Game mode is set to 1 player
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 4
USER_CLR: WHITE
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 4
USER_CLR: WHITE
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
white player - enter your move:
Restarting...
Specify game setting or type 'start' to begin a game with the current setting:
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
Wrong game mode
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
Wrong difficulty level. The value should be between 1 to 5
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
Invalid command
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
Game mode is set to 1 player
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 3
USER_CLR: WHITE
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 3
USER_CLR: BLACK
SETTINGS:
GAME_MODE: 1
DIFFICULTY_LVL: 2
USER_CLR: WHITE
Invalid command
Invalid command
Invalid command
Invalid command
Invalid command
Invalid command
Invalid command
Invalid command
Invalid command
Invalid command
Exiting...
